src/main/ts/components/Editor.tsx

```diff
@@ -192,6 +209,6 @@ export class Editor extends React.Component<IAllProps> {
       this.elementRef.current.style.visibility = '';
     }
 
-    getTinymce().init(finalInit);
+    getTinymce()?.init(finalInit);
```

Comment:
I think we might need to update version of TypeScript in `package.json` to 3.7 to allow for Optional Chaining.

Reply:
Done (now typescript 4)